recipes/ndf/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "ndf" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a87c0bd2cccbc438d24859cbe64de34157b432f104d0e7b59b0610264b538522
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py2k]
```

This looks like it could be a [noarch package](https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-builds):
```suggestion
  noarch: python
```

---

Currently it does not support python2. There are some Numpy operations that are not supported in Python2.7

---

Now I see. Agree with a change

---

Sorry I should have suggested `python >=3` for the host and run requirements.

---

I have changed it. I should notice yesterday, but it was late xD